Sources/KlaviyoSwift/KlaviyoSwiftEnvironment.swift

```diff
@@ -31,7 +31,7 @@ struct KlaviyoSwiftEnvironment {
             stateChangePublisher: StateChangePublisher().publisher,
             setBadgeCount: { count in
                 Task {
-                    if let userDefaults = UserDefaults(suiteName: Bundle.main.object(forInfoDictionaryKey: "Klaviyo_App_Group") as? String) {
+                    if let userDefaults = UserDefaults(suiteName: Bundle.main.object(forInfoDictionaryKey: "klaviyo_app_group") as? String) {
```

Comment:
i see below theres a guard,
```
            guard let appGroup = Bundle.main.object(forInfoDictionaryKey: "klaviyo_app_group") as? String,
                  let userDefaults = UserDefaults(suiteName: appGroup) else {
                return
            }
```
should that also be done here in case we end up with `UserDefaults(suiteName: nil)`?

Reply:
and if that is the case, is it worth pulling into a util function somewhere?

Reply:
if let is basically the same as a guard, it handles it carefully, just personal preference on readability ig, can switch to the guards if thats preferable

Reply:
nice thats fine then i think, seems like guard returns in the indent whereas if let executes in the indent so its a matter of where you want nesting vs early returns

Reply:
since this is a large block in the execute, i would opt for an early return vs the other file where the execution is a single like, the if let reads better there

Reply:
but i dont feel strongly